docs/user-guide/kubectl/kubectl_exec.md

```diff
@@ -62,7 +62,6 @@ $ kubectl exec 123456-7890 -c ruby-container -i -t -- bash -il
 
 ```
   -c, --container="": Container name. If omitted, the first container in the pod will be chosen
-  -p, --pod="": Pod name
```

Comment:
Just verifying since this line is removed, is ```--pod``` also depreciated for ```exec```? It seems to be listed above in ```docs/man/man1/kubectl-exec.1```